From 743ce23fbcec9153deb6ca1c16858a9a62554676 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Mon, 19 Aug 2019 21:13:52 +0200
Subject: [PATCH 059/152] util: Separate statements with linebreak

Our eslint rules will soon forbid more than one statement per line.
We already follow that rule except for one lone offender; fix that.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/710
---
 js/misc/util.js | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/js/misc/util.js b/js/misc/util.js
index d8ee946ee..87a6c1225 100644
--- a/js/misc/util.js
+++ b/js/misc/util.js
@@ -313,7 +313,8 @@ function lowerBound(array, val, cmp) {
     if (array.length == 0)
         return 0;
 
-    min = 0; max = array.length;
+    min = 0;
+    max = array.length;
     while (min < (max - 1)) {
         mid = Math.floor((min + max) / 2);
         v = cmp(array[mid], val);
-- 
2.23.0

